recipes-kernel/kernel-modules/x8h7/x8h7_can.c

```diff
@@ -30,7 +30,7 @@
 #define DRIVER_NAME "x8h7_can"
 
 /* DEBUG HANDLING */
-// #define DEBUG
+ #define DEBUG
```

Comment:
I'd avoid merging with DEBUG on all the time (could really harm the entire board operation)

Reply:
That's why it came as a separate commit - easy to revert ;)

Reply:
Dropped it via rebase.